packages/indexer-client/src/dataMappers.ts

```diff
@@ -431,7 +437,9 @@ export function mapIndexerMarketSnapshot(
     ),
     cumulativeInflows: mapValues(snapshot.cumulative_inflows, toBigDecimal),
     cumulativeOutflows: mapValues(snapshot.cumulative_outflows, toBigDecimal),
-    oraclePrices: mapValues(snapshot.oracle_prices, fromX18),
+    oraclePrices: mapValues(snapshot.oracle_prices, (value) =>
```

Comment:
Could this just be `mapValues(snapshot.oracle_prices, removeDecimals)`?

Reply:
Same comment with above

Reply:
`removeDecimals` hasa  second parameter which is decimals